### ISSUE TITLE
possibility to configure optimizations with context manager?

### ISSUE THREAD

**Issue by lgray** [dask-contrib/dask-awkward#548]
It would be nice to have a configuration option for optimizations as in dask-array and dask-delayed:
https://docs.dask.org/en/stable/optimize.html#customizing-optimization

**Comment by martindurant** [comment]
We do: https://github.com/dask-contrib/dask-awkward/blob/main/src/dask_awkward/awkward.yaml#L16 . Obviously there could be more.

**Comment by lgray** [comment]
Yes, in particular the ability to completely override the optimizations with a given function would be great. :-)
